Description: Prefer "default.desktop", so that other desktop environments like XFCE can use gdm, too, without being forced to start GNOME.
Ubuntu: https://launchpad.net/bugs/403291
Upstream: https://bugzilla.gnome.org/show_bug.cgi?id=594733
Index: gdm-3.14.2/daemon/gdm-session.c
===================================================================
--- gdm-3.14.2.orig/daemon/gdm-session.c	2015-07-09 13:06:06.447523437 +0200
+++ gdm-3.14.2/daemon/gdm-session.c	2015-07-09 13:06:06.447523437 +0200
@@ -546,7 +546,7 @@
                 }
         }
 
-        name = g_strdup ("gnome");
+        name = g_strdup ("default");
         if (get_session_command_for_name (name, NULL)) {
                 g_free (self->priv->fallback_session_name);
                 self->priv->fallback_session_name = name;
